services/chainstorage/src/main/java/tech/pegasys/teku/services/chainstorage/StorageService.java

```diff
@@ -92,7 +90,10 @@ protected SafeFuture<?> doStart() {
               try {
                 database = dbFactory.createDatabase();
               } catch (EphemeryException e) {
+                final EphemeryDatabaseReset ephemeryDatabaseReset = new EphemeryDatabaseReset();
                 database = ephemeryDatabaseReset.resetDatabaseAndCreate(serviceConfig, dbFactory);
+                LOG.warn(
+                    "Ephemery network deposit contract id has updated, resetting the stored database and slashing protection data.");
```

Comment:
these sorts of things we want to print prior to the logic in case this is what kills the system, then we still have the context...